### PR TITLE
Drop saved_queries table

### PR DESCRIPTION
**Pull request by daxmc99**
We no longer use this table

### CODE REVIEW

File: migrations/1528395719_migration_sql.down.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+BEGIN;
+CREATE TABLE IF NOT EXISTS  saved_queries
+(
+    query text not null,
+    last_executed timestamp with time zone not null,
+    latest_result timestamp with time zone not null,
+    exec_duration_ns bigint not null
+);
+
+alter table saved_queries owner to sourcegraph;
```
Comment (asdine) [inline migrations/1528395719_migration_sql.down.sql:10]: super nit: All the existing sql migration scripts seem to use uppercase for all SQL keywords, I think we should do the same for consistency

File: migrations/1528395719_migration_sql.down.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+BEGIN;
+CREATE TABLE IF NOT EXISTS  saved_queries
```
Comment (asdine) [inline migrations/1528395719_migration_sql.down.sql:2]:
```suggestion
CREATE TABLE IF NOT EXISTS saved_queries
```

**Review by asdine — APPROVED** [review]

File: migrations/1528395719_migration_sql.down.sql (hunk 1 above)
Comment (daxmc99) [reply migrations/1528395719_migration_sql.down.sql]: Updated 👍

**Review by unknwon — APPROVED** [review]

**Comment by unknwon** [comment]
I think you also need to run `go generate ./internal/db/` to update the "schema.md" (that's why CI failed).

**Review by ryanslade — APPROVED** [review]

**Pull request merged**